packages/musl.rb

```diff
@@ -0,0 +1,40 @@
+require 'package'
+
+class Musl < Package
+  description 'A modern, simple, and fast C library implementation that strives to be lightweight, fast, simple, free, and correct in the sense of standards-conformance and safety.'
+  homepage 'https://www.musl-libc.org/'
+  version '1.1.21'
+  source_url 'https://www.musl-libc.org/releases/musl-1.1.21.tar.gz'
+  source_sha256 'c742b66f6f49c9e5f52f64d8b79fecb5a0f6e0203fca176c70ca20f6be285f44'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  def self.build
+    abi = 'eabihf' if ARCH == 'armv7l'
```

Comment:
Probably should add `abi = ''` above this line so other architectures have this variable assigned.

Reply:
It works.

Reply:
It works but it's not good practice.  You could potentially get unexpected results using a variable that is unassigned.

Reply:
OK.